ipn/ipnlocal/local_test.go

```diff
@@ -3766,10 +3776,79 @@ func TestLocalBackendSuggestExitNode(t *testing.T) {
 			},
 			wantErr: ErrCannotSuggestExitNode,
 		},
+		{
```

Comment:
I'd recommend one additional test that has the allowed suggested exit nodes as a not-`nil` list of length 0, which signals "nothing is allowed as a suggestion".